trashtalk/google_sheets.py

```diff
@@ -8,17 +8,16 @@
 from oauth2client.service_account import ServiceAccountCredentials
 
 from trashtalk.models import Cleanup, db_session
-from trashtalk.master_keys import MasterKeys
-key_chain = MasterKeys()
+from flask import Blueprint, render_template, request, flash, current_app
 
 scope = ['https://spreadsheets.google.com/feeds/'] #Sheet must be shared with Cloud account in order to find it
-validation =key_chain.validation #Validation pulls from JSON account information file
-sheet_key = key_chain.sheet_key #Key from sheet URL
+validation = ""#Validation pulls from JSON account information file
```

Comment:
We dont seem to need any of these variables any more. Let's delete them.
```
validation
sheet_key
```

Reply:
The values themselves are still necessary. However, they can be set with **settings.py/dev.cfg**. I'm just not sure how to do this. In my version of the code, almost all variables defined in **settings.py/dev.cfg** are broken. I'll move these variables fully to **settings.py/dev.cfg**. But, I need help getting them to properly function.